tap_github/client.py

```diff
@@ -405,5 +405,12 @@ def calculate_sync_cost(
     ) -> Dict[str, int]:
         """Return the cost of the last graphql API call."""
         costgen = extract_jsonpath("$.data.rateLimit.cost", input=response.json())
-        cost = next(costgen)
+        try:
+            cost = next(costgen)
```

Comment:
How about using a default value like this?

https://stackoverflow.com/questions/14250184/default-value-for-next-element-in-python-iterator-if-iterator-is-empty

Reply:
Nice! I didn't know `next` could do that.